### PR TITLE
Update Documentations

### PR DESCRIPTION
**Pull request by hungdche**
Address issue #358
- [x] Camera Documentations
- [ ] How to run apps
- [x] How to use record_imu_data 

Close issue #355 
- [ ] No mention of offloading
- [ ] Instrutions on how to generate APE/RPE with eco
- [x] Update links: ZED Mini, release, chat (Discord instead of Gitter)
- [x] Update talks/papers to match illixr.org
- [x] No mention of the consortium meetings, possible links old meetings as well
- [x] Change the documentation to reflect the separated imu and cam

### CODE REVIEW

**Comment by hungdche** [comment]
@jianxiapyh @qinjunj I would need help updating docs for these two checkboxes:

> No mention of offloading
> Instrutions on how to generate APE/RPE with eco

**Comment by qinjunj** [comment]
> @jianxiapyh @qinjunj I would need help updating docs for these two checkboxes:
> 
> > No mention of offloading
> > Instrutions on how to generate APE/RPE with eco

Will be addressed in #392.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![NCSA licensed](https://img.shields.io/badge/license-NCSA-blue.svg)](LICENSE)
 [![CI](https://github.com/ILLIXR/ILLIXR/workflows/illixr-tests-master/badge.svg)](https://github.com/ILLIXR/ILLIXR/actions)
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/ILLIXR/community)
+![Discord](https://img.shields.io/discord/830812443189444698)
```
Comment (Jebbly) [inline README.md:5]: I think this might be missing a link to our actual Discord, would that be possible?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ You can find the complete ILLIXR system [here][38].
 
 ILLIXR also provides its components in standalone configurations to enable architects and
     system designers to research each component in isolation.
-The standalone components are packaged together in the [v1-latest release][39] of ILLIXR. 
+The standalone components are packaged together in the [v3.1.0 release][39] of ILLIXR.
```
Comment (Jebbly) [inline README.md:28]: Let's say something like "as of v3.x.x" or something, so we don't have to update this every version update.

File: docs/using_cameras.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+1. **Install ZED SDK** 
+
+    Install the latest version of the ZED SDK on [stereolabs.com][1].<br>
+    For more information, checkout the [ZED API documentation][2].
+
+2. **Get ZED's calibration parameters**
+
+    Both OpenVINS and Kimera have a decent calirbation parameters for ZED. But if you wish to add your own: 
+    
+        ./usr/local/zed/tools/ZED_Calibration
```
Comment (Jebbly) [inline docs/using_cameras.md:38]: I think this `.` might be a typo?

File: docs/using_cameras.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    
+        ./PATH/TO/LIBREALSENSE/build/tools/enumerate-devices 
+
+    Run this command to obtain the calibration parameters
+
+        ./rs-enumerate-devices -c
+
+3. **Run ILLIXR with RealSense:**  
+
+    Uncomment `realsense` in `configs\native` and run ILLIXR normally.
```
Comment (Jebbly) [inline docs/using_cameras.md:70]: `configs\native` -> `configs/native.yaml`

File: docs/using_cameras.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+    Your original factory calibration file is stored here
+
+        /usr/local/zed/settings/
+
+    Or download it from [calib.stereolabs.com][8].
+
+3. **Run ILLIXR with ZED:**  
+
+    Uncomment `zed` in `configs\native` and run ILLIXR normally.
```
Comment (Jebbly) [inline docs/using_cameras.md:48]: `configs\native` -> `configs/native.yaml`

**Review by Jebbly — DISMISSED** [review]
LGTM, I will be adding some commits (at least one to fix a fauxpose doc typo), and more documentation will be incoming with future PRs.

**Review by astro-friedel — APPROVED** [review]
These updates look good, but will need to be tweaked once the new build system is deployed.

File: README.md (hunk 1 above)
Comment (hungdche) [reply README.md:5]: I would need a Discord admin to get the server ID (as specified [here](https://github.com/dgibbs64/discord-banners/blob/master/README.md))

Comment (jianxiapyh) [reply README.md:5]: I am not sure, but I just checked the server ID is 830812443189444698 which is the same as the one mentioned in this current version of documentation

Comment (jianxiapyh) [reply README.md:5]: Discord
so something like this?

Comment (jianxiapyh) [reply README.md:5]: does this work?
`Discord`

File: docs/using_cameras.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+    Your original factory calibration file is stored here
+
+        /usr/local/zed/settings/
+
+    Or download it from [calib.stereolabs.com][8].
+
+3. **Run ILLIXR with ZED:**  
+
+    Uncomment `zed` in `configs/native.yaml` and run ILLIXR normally.
```
Comment (jianxiapyh) [inline docs/using_cameras.md:48]: You also need to update the flag in VIO right?

Comment (jianxiapyh) [reply docs/using_cameras.md:48]: zed also does not exist in configs/native.yaml

File: docs/using_cameras.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+    
+        ./PATH/TO/LIBREALSENSE/build/tools/enumerate-devices 
+
+    Run this command to obtain the calibration parameters
+
+        ./rs-enumerate-devices -c
+
+3. **Run ILLIXR with RealSense:**  
+
+    Uncomment `realsense` in `configs/native.yaml` and run ILLIXR normally.
```
Comment (jianxiapyh) [inline docs/using_cameras.md:70]: same questions as zed

File: record_imu_cam/README.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    
+        timestamp [ns],w_x [rad s^-1],w_y [rad s^-1],w_z [rad s^-1],a_x [m s^-2],a_y [m s^-2],a_z [m s^-2]
+
+## How to rerun recorded dataset
+
+1. **(IMPORTANT)** In `configs/rt_slam_plugins.yaml`, comment this line: 
+
+       # - path: record_imu_cam/  
+
+2. In `configs/native.yaml` (or whatever mode you're running ILLIXR with), add the path of the recorded dataset like so:
```
Comment (jianxiapyh) [inline record_imu_cam/README.md:43]: I don't know if this is how we should use data variable as  input and output variable path.

**Review by jianxiapyh — CHANGES_REQUESTED** [review]
see my comment

File: README.md (hunk 1 above)
Comment (hungdche) [reply README.md:5]: I think you might have to enable "widget" on Discord ([ref](https://github.com/FriiZoLoGYy/Discord-Github-Badges))

File: record_imu_cam/README.md (hunk 8 above)
Comment (hungdche) [reply record_imu_cam/README.md:43]: That part is for running the recorded dataset that is captured from the `record_imu_cam` plugin. The default output path is `ILLIXR/`

File: docs/using_cameras.md (hunk 6 above)
Comment (hungdche) [reply docs/using_cameras.md:48]: You're right. It's in `configs/rt_slam_plugins/yaml`. My bad.

File: README.md [hunk 9]
```diff
@@ -76,10 +76,14 @@ A [talk presented at NVIDIA GTC'21][42] describing ILLIXR and announcing the ILL
 
 A [demo][45] of an OpenXR application running with ILLIXR.
 
+For more up-to-date list of related papers, demos, and talks, please visit [illixr.org][37].
+
 The [ILLIXR consortium][37] is an industry-supported community effort to democratize
     XR systems research, development, and benchmarking.
 Visit our [web site][37] for more information.
 
+The ILLIXR consortium is also holding a biweekly consortium meeting. For past meetings, for more information, past meeting recordings, and request for presenting, please visit [here][50].
```
[the remark this answers is not in the document]
Comment (hungdche) [reply README.md:85]: I will just add refer to Discord for announcement so we don't have to update README when we restarted the meetings.

[the remark this answers is not in the document]
Comment (jianxiapyh) [reply README.md]: sounds good

File: README.md (hunk 1 above)
Comment (jianxiapyh) [reply README.md:5]: I just enabled server widget. Give it a try now

**Review by jianxiapyh — COMMENTED** [review]
Just let me know if the widget now works or not then I can approve the changes

**Comment by hungdche** [comment]
@jianxiapyh The discord shield is working now.
image

**Review by jianxiapyh — APPROVED** [review]
LGTM

File: mkdocs.yml [hunk 10]
```diff
@@ -7,6 +7,7 @@ nav:
         - 'ILLIXR': index.md
         - 'Getting Started': getting_started.md
         - 'Building ILLIXR': building_illixr.md
+        - 'Using ILLIXR with Cameras': using_cameras.md
         - 'ILLIXR Debugging Tips': debugging_illixr.md
         - 'ILLIXR Under Virtualization': virtualization.md
         - 'ILLIXR Plugins': illixr_plugins.md
```
Comment (qinjunj) [inline mkdocs.yml:13]: Include docs to `offload_vio` somewhere in this file. That specific doc will be updated in #405. LGTM after the change.

**Review by qinjunj — COMMENTED** [review]
May need a rebase and can be merged last.

File: README.md [hunk 11]
```diff
@@ -2,7 +2,8 @@
 
 [![NCSA licensed](https://img.shields.io/badge/license-NCSA-blue.svg)](LICENSE)
 [![CI](https://github.com/ILLIXR/ILLIXR/workflows/illixr-tests-master/badge.svg)](https://github.com/ILLIXR/ILLIXR/actions)
```
Comment (hungdche) [inline README.md:4]: Need someone who can login into Jenkins to update the status of CI (ref: https://shields.io/badges/jenkins). I.e. I would need the Job URL (Example: https://wso2.org/jenkins/view/All%20Builds/job/archetypes)

Comment (astro-friedel) [reply README.md:4]: I could probably do that, but I do not have access to the example web page, so I do not know what particulars are needed.

File: docs/using_cameras.md
Comment (astro-friedel) [inline docs/using_cameras.md:1]: The Kimera-VIO stuff should be removed as we are dropping support for it.

File: README.md (hunk 11 above)
Comment (hungdche) [reply README.md:4]: I could not log into Jenkins as well. @shg8 can you help me get the link to the Jenkins build?

File: docs/using_cameras.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
+
+# Using Cameras with ILLIXR
+
+ILLIXR supports a wide range of cameras, most of which are for the purpose of feeding images and IMU measurements to the system. <br>
+*To learn more about how it works, checkout [Building ILLIXR][12].* <br>
+*If you are interested in what topics these cameras feed into, checkout [ILLIXR plugins][10].*
+
+
+**Important Note**: <br>
+Before running any of these plugins below, it is important to comment out `offline_imu_cam` in `configs\native.yaml`. <br>
```
Comment (jianxiapyh) [inline docs/using_cameras.md:10]: I am assuming we are using offline_cam and offline_imu now?

File: docs/using_cameras.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+
+    This step is only required if using OpenVINS. Uncomment this [line](9) in the OpenVINS plugin. 
+
+4. **Run ILLIXR with ZED:**  
+
+    Uncomment `zed` in `configs/rt_slam_plugins.yaml` and run ILLIXR normally.
+
+## Intel Realsense 
+
+ILLIXR has been tested with Inteal RealSense D455, but it should work with any D or T series RealSense Camera.
```
Comment (jianxiapyh) [inline docs/using_cameras.md:50]: OpenVINs parameter needs to be updated to work properly. Say that we will release the corresponding OpenVINs flag (#define realsense) soon.

File: docs/using_cameras.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+
+    Your original factory calibration file is stored here
+
+        /usr/local/zed/settings/
+
+    Or download it from [calib.stereolabs.com][8].
+
+3. **Enable ZED in OpenVINS plugin**
+
+    This step is only required if using OpenVINS. Uncomment this [line](9) in the OpenVINS plugin.
```
Comment (jianxiapyh) [inline docs/using_cameras.md:42]: link doesn't work

**Review by jianxiapyh — DISMISSED** [review]

**Review by jianxiapyh — APPROVED** [review]

**Review by astro-friedel — APPROVED** [review]

File: mkdocs.yml (hunk 10 above)
Comment (astro-friedel) [reply mkdocs.yml:13]: The offload_vio documentation is already in master and cmake branches I believe.

**Review by astro-friedel — APPROVED** [review]

**Review by jianxiapyh — APPROVED** [review]

**Pull request merged**